test/fetching_books_and_highlights_test.rb

```diff
@@ -101,6 +106,10 @@ def raw_quotes_page
                     Destiny is not born of decision; it is born of uncontrollable circumstances.
                   </span>
                 </div>
+                <div id="note-" class="a-row a-spacing-top-base kp-notebook-note aok-hidden kp-notebook-selectable">
+                  <span id="note-label" class="a-size-small a-color-secondary">Note:</span>
+                  <span id="note" class="a-size-base-plus a-color-base">This is a note!</span>
```

Comment:
I know this is super-picky, but I like to see test data that mirrors something I'd actually use. It doesn't have to be very involved, something like:
```html
 <span id="note" class="a-size-base-plus a-color-base">This was an inspiring quote.</span>
```
would suffice.